pkg/services/reputation/common/managers/managers.go

```diff
@@ -12,6 +14,15 @@ import (
 	"go.uber.org/zap"
 )
 
+type ErrNoSuchPeerInNetmap struct {
```

Comment:
Do we need to export that, though?

Reply:
not for now, i think, but maybe it could be useful in the future

make it private?

Reply:
Make it public when it's needed. Unlike `neofs-api-go`, it is not a library, so we are in control of this error usage.

Reply:
got it, changed